sympy/stats/crv_types.py

```diff
@@ -1177,7 +1177,14 @@ def check(a, b):
     def pdf(self, x):
         a, b = self.a, self.b
         return b**a/gamma(a) * x**(-a-1) * exp(-b/x)
-
+    def sample(self):
+	try :
+	     from scipy.stats import invgamma
```

Comment:
It looks like you are using tabs here. Be sure that you configure your editor to use four spaces for indentation.